concepts/basics/introduction.md

```diff
@@ -22,18 +30,19 @@ int tomatoes{80};
 int potatoes = 40;
 ```
 
-## Arithmetic Operations
+```exercism/caution
+C++ does allow using uninitialized variables.
+Until the variable is deliberately set, it is undefined and might contain anything.
+To avoid used-before-set errors and undefined behaviour it is adviseable to **always initialize**.
```

Comment:
the word `behaviour` is the British spelling, and the standard uses the American spelling `behavior`.
https://en.cppreference.com/w/cpp/language/ub